src/components/CheckBoxTree/style.css

```diff
@@ -1,91 +1,102 @@
+:root {
+    --text-label-margin: 8px;
+}
+
 .container {
     background-color: var(--dark);
 }
 
-.sub-menu {
-    flex-flow: row;
+.header-label {
+    margin-left: var(--text-label-margin);
+    max-width: 120px;
+    font-weight: 200;
 }
 
-.sub-menu :global(.ant-col-offset-3) {
-    margin-left: 13.4%;
+.container :global(.ant-typography-ellipsis) {
+    color: var(--dark-theme-sidebar-text)
 }
 
-.header-label {
-    margin-left: 8px;
-    max-width: 120px;
+.container :global(.ant-checkbox-inner) {
+    border-color: var(--warm-gray);
+}
+
+.container :global(.ant-checkbox-indeterminate .ant-checkbox-inner::after) {
+    background-color: var(--dark-theme-header-bg);
+    height: 2px;
 }
 
-.checkbox-set {
-    width: 53px;
+.check-all-checkbox {
+    width: 100%;
+    margin-left: 88px; /* to line up with the checkboxes below */
+    padding: 10px 0;
 }
 
-.checkbox-set :global(.ant-checkbox) {
-    top: 12px;
+.check-all-checkbox :global(.ant-checkbox) {
+    margin-right: var(--text-label-margin);
 }
 
-.container .checkbox-labels {
-    display: flex;
-    flex-direction: column;
+.no-children-row {
+    margin-left: 32px; /* keeps it in the same position as rows with dropdowns */
 }
 
-.row-label-container {
-    display: inline-block;
+.no-children-row :global(.ant-checkbox-wrapper) {
+    width: 30px;
     height: 40px;
-    padding: 2px 0;
 }
 
-.row-label {
-    margin-left: 8px;
-    font-weight: 200;
-    vertical-align: text-top;
-    display: inline-block;
-    height: 16px;
-    line-height: 1;
+.star-icon::after {
+    content: "\e902";
+    color: var(--dark-theme-sidebar-text);
 }
 
-.container .col-labels {
-    background-color: var(--dark-theme-sidebar-item-bg);
-    margin-bottom: 3px;
+.container :global(.header-checkbox .ant-checkbox) {
+    top: 3.5px;
 }
 
-.container .col-labels label {
-    font-size: 10px;
-    text-transform: uppercase;
+.sub-menu {
+    flex-flow: row;
 }
 
-.container :global(.ant-checkbox-inner) {
-    border-color: var(--warm-gray);
+.sub-menu :global(.ant-col-offset-3) {
+    margin-left: 13.4%;
```

Comment:
Is this an arbitrary value?

Reply:
it is, but it's actually not new. I think this was a way for me to do consistent columns that were between ant's 3 and 4.